book/src/users/problems.md

```diff
@@ -0,0 +1,16 @@
+# Problems
+
+If you encounter a problem, keep in mind that Trin is under active development.
+Some issues may be lower on the priority list.
+
+## Search for more information
+
+Try searching:
+- This book
+- The Trin repository issues
+
+## Document the problem
+
+If the problem seems new, raise an issue in the Trin repository.
```

Comment:
I would include a link to the [issues](https://github.com/ethereum/trin/issues) page